src/main.rs

```diff
@@ -2,9 +2,17 @@ use geo::Point;
 use serde_json;
```

Comment:
Some style for imports (the way you have it isn't incorrect, just not really standard, and unfortunately cargo fmt doesn't autofix these):

1) You don't need to explicitly import a crate; your Cargo.toml declares it. So you can remove `use serde_json;` here.
2) The `{}` notation is only for importing multiple things from somewhere, so it should just be `use serde_json::to_string_pretty`
3) You can use `{}` to import multiple things from one place. So `use zonebuilder::{clockboard, Params};`